LibDeflate.lua

```diff
@@ -2762,7 +2766,12 @@ local function Inflate(state)
 		end
 		if status ~= 0 then
 			return nil, status
-		end
+        end
+        -- ComputerCraft requires this for long-running processes
+        if os and os.pullEvent then
```

Comment:
Don't think this is needed.
If the input string is too long. Caller can split the string into multiple parts and call LibDeflate:Compress multiple times and concat the input string.